configuration/config-txt/video.md

```diff
@@ -663,7 +663,7 @@ If using the VC4 FKMS V3D driver (this is the default on the Raspberry Pi 4), th
 
 ### display_lcd_rotate
 
-Use `display_lcd_rotate` to rotate or flip the LCD orientation. Parameters are the same as `display_hdmi_rotate`.
+For the legacy graphics driver (default on models prior to the Pi4), use `display_lcd_rotate` to rotate or flip the LCD orientation. Parameters are the same as `display_hdmi_rotate`. For the modern graphics driver use `lcd_rotate`.
```

Comment:
That's not quite correct - `lcd_rotate` can be used with any of the 3 possible graphics drivers. Also, the documentation does not currently specify what options are available for `lcd_rotate` - it only gives an example of one possible option.

Reply:
Per [this page of the documentation](https://www.raspberrypi.org/documentation/configuration/display_rotation.md), `display_lcd_rotate` is a legacy driver property. Note that there is a difference between `lcd_rotate`, `display_lcd_rotate` and `display_rotate` 😆 🔥 
 
![legacy](https://user-images.githubusercontent.com/8799628/86317031-047df780-bbf4-11ea-8762-294bc3a63059.png)

Reply:
@JamesH65 will no doubt be able to confirm, but from reading the current docs it sounds like `display_rotate` (which is labelled as deprecated), `display_lcd_rotate` and `display_hdmi_rotate` options apply at the GPU-driver level, and are thus only applicable to the legacy graphics driver; whereas `lcd_rotate` works at the LCD-screen hardware level, and is thus applicable to all graphics drivers. So `lcd_rotate` is actually an orthogonal option to the others, and not a "replacement" for `display_lcd_rotate` ? :shrug: 

@JamesH65 As noted by @andrum99 , it's not clear what the valid values for `lcd_rotate` are - does it accept _all_ of the values that `display_hdmi_rotate` does, or only a subset? Perhaps `lcd_rotate` should also be documented on the `display_rotation.md` page, and then all the `*_rotate` options on `video.md` could just refer to `display_rotation.md` ?

Reply:
IIRC, lcd_rotate does indeed talk direct to the LCD which does the rotate itself (although it's more like horizontal and vertical flips rather than rotate). The other options use the transposer to do rotation - which is only available when using firmware -  so legacy and I think FKMS, but not sure what nasty interactions will occur in FKMS mode.  There's also no guarantee that any future LCD's will be able to use that lcd_rotate option as its part of the LCD hardware.

Reply:
My personal preference for this PR would be to reword the final sentence like this: 'See also `lcd_rotate`.' The rest looks good to me 😁 Improving the documentation for `lcd_rotate` could be tackled in a separate PR.

Reply:
@andrum99 I have updated the PR to use the `See also lcd_rotate` phrasing like you suggested 👌

Reply:
LGTM 😁